File: 351004/Brazhalovich/Lab_1/discussion/src/main/java/org/example/discussion/entity/Comment.java
package org.example.discussion.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.cassandra.core.mapping.Column;
import org.springframework.data.cassandra.core.mapping.PrimaryKey;
import org.springframework.data.cassandra.core.mapping.Table;

@Table("tbl_comment")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Comment {
    @PrimaryKey
    private Long id;
    @Column("news_id")
    private Long newsId;
    @Column("user_id")
    private Long userId;
    private String content;
    private String state;
}

File: 351002/Khmel/Task330/Discussion/Models/Comment.cs
using Cassandra.Mapping.Attributes;

namespace Discussion.Models
{
    [Table("tbl_comment", Keyspace = "distcomp")]
    public class Comment
    {
        [PartitionKey]
        [Column("id")]
        public long Id { get; set; }

        [Column("story_id")]
        public long StoryId { get; set; }

        [Column("content")]
        public string Content { get; set; } = "";
    }
}